src/icons/IconBase copy.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef, ReactNode } from 'react';
+
```

Comment:
This component filename is `IconBase copy.tsx`

I'm guessing it is a file that needs to be removed or renamed. 🤓